adapters/balance.js

```diff
@@ -38,6 +38,14 @@ module.exports = function (mu, transports) {
 
 
 
+  function setMu (muInstance) {
+    transports.forEach(function (transport) {
+      transport.setMu(muInstance)
+    })
+  }
```

Comment:
The mu instance is in the closure scope which is why the setter is required. Outbound does require access to the mu instance!